packs/spell-effects/spell-effect-dragon-wings.json

```diff
@@ -1,10 +1,10 @@
 {
-    "_id": "jy4edd6pvJvJgOSP",
+    "_id": "ZPfRFrixTwTSlbYe",
```

Comment:
Consider keeping the old id for this one, and redirecting the other.

Reply:
Makes sense.